source/DistanceAndDirection/DistanceAndDirectionLibrary/Views/CircleView.xaml

```diff
@@ -101,7 +107,13 @@
                           Margin="3,3,0,0"
                           ItemsSource="{Binding Source={StaticResource lineDistanceData}}"
                           SelectedItem="{Binding Path=LineDistanceType,
-                                                 Mode=TwoWay}" />
+                                                 Mode=TwoWay}" >
+                    <ComboBox.ItemContainerStyle>
+                        <Style TargetType="ComboBoxItem">
+                            <Setter Property="Foreground" Value="Black" />
```

Comment:
The ProSDK has some items like `{DynamicResource=Esri_GrayXXX}` that are meant to work for both themes - so switching to these values might be a longer-term plan